cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -310,12 +310,19 @@ message TraceBounds {
 message SegYSeismicRequest {
     oneof identifier {
         Identifier seismic = 1;
-        int64 seismic_store_id = 2;
+        int64 seismic_store_id = 2;  // Need to be a data manager or tenant user, not 3rd party, to access by tracestore
     }
+    // This only supports queries to 3D objects.
+    // Deprecated. Use filter instead.
     oneof query {
         Geometry polygon = 3;
         LineBasedRectangle lines = 4;
     }
```

Comment:
Hmm, these fields include a `z_range`. Is that silently ignored, currently? If so, the new field is an improvement :)

Reply:
The `RawTraceStreamConfig` struct doesn't have a `z_range` filter so I imagine we're just silently ignoring it